### ISSUE TITLE
Make snapshot folder configurable for clean snapshots

### ISSUE THREAD

**Issue by hegedusm** [cerner/terra-toolkit-boneyard#81]
# Feature Request

## Description
cleanSnapshots scans the whole project for snapshot folders. This also includes the node_modules directory. This clean and scan step takes a significant amount of time during running wdio tests locally. Adding an optional configuration that sets where these folders should be located in the project (e.g. "tests") reduces the wait time.

## Additional Context / Screenshots
Clean is called during wdio execution. Scan pattern contains 'process.cwd()/**/'.

**Comment by hegedusm** [comment]
Partially done in https://github.com/hegedusm/terra-toolkit-boneyard/tree/configurable-snapshot-clean-directory

**Comment by ryanthemanuel** [comment]
Hi @hegedusm. We have recently made the cleaning of snapshots a lot easier with our new terra-functional-testing package. I would strongly recommend upgrading when you have the time as we have improved a lot of things as well. With the new package, terra-toolkit-boneyard is in maintenance mode and will only be modified to fix major issues, so I'm going to go ahead and close this.
